src/rabbit_channel.erl

```diff
@@ -1726,8 +1818,8 @@ record_sent(ConsumerTag, AckRequired,
                         conn_name         = ConnName,
                         channel           = ChannelNum}) ->
     ?INCR_STATS(queue_stats, QName, 1, case {ConsumerTag, AckRequired} of
-                                           {none,  true} -> get;
-                                           {none, false} -> get_no_ack;
+                                           {_,  true} when is_integer(ConsumerTag) -> get;
+                                           {_, false} when is_integer(ConsumerTag) -> get_no_ack;
```

Comment:
Looking at the code I don't think that guard is right, I wonder if something changed while implementing it and that's a leftover. @kjnilsson any idea why this guard would be there?

Reply:
The git-blame commit 45a7bac81ba53552a3f78eea05b3d0b316b71b02 implies that the integer consumer tag replaced the `none` atom consumer tag for `basic.get`. It can be because `basic.get` is not a part of the raft machine API and is emulated with one-off consumer.

Reply:
Clarified in the comments: https://github.com/rabbitmq/rabbitmq-server/pull/1706/commits/f10ec8d09df4a94bb8daaf93435e8362cce2ddea